### PR TITLE
student model controller view

### PR DESCRIPTION
**Pull request by mikeinflunity**
Created a student model

added a controller and a #new view.

I drafted also the relation with students group but still thinking how to solve that

### CODE REVIEW

File: app/models/group.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Group < ApplicationRecord
+  belongs_to :students_group
```
Comment (arieivs) [inline app/models/group.rb:2]: the other way around, the group has_many :students_group

File: app/models/students_group.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class StudentsGroup < ApplicationRecord
+  belongs_to :student
+  has_many :groups
```
Comment (arieivs) [inline app/models/students_group.rb:3]: belongs_to :groups
connecting tables have only belongs_to

File: db/migrate/20200630120116_create_groups.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateGroups < ActiveRecord::Migration[6.0]
+  def change
+    create_table :groups do |t|
+      t.string :class_name,
+
```
Comment (arieivs) [inline db/migrate/20200630120116_create_groups.rb:5]: missing t.references :teachers, I get you didn't do it before we had the teachers, maybe we should have waited for that! so don't forget adding a migration for it once you have our teachers model in you pc

File: db/schema.rb [hunk 4]
```diff
@@ -10,11 +10,27 @@
...
+ActiveRecord::Schema.define(version: 2020_06_30_120116) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "groups", force: :cascade do |t|
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.string "class_name"
+    t.string "#<ActiveRecord::ConnectionAdapters::PostgreSQL::TableDefinition"
```
Comment (arieivs) [inline db/schema.rb:22]: what's this? something went wrong

Comment (arieivs) [reply db/schema.rb:22]: I see you made models for student, student_group and group, but you only made migrations for student and group. I think this is the origin of this strange last line

File: app/models/group.rb (hunk 1 above)
Comment (mikeinflunity) [reply app/models/group.rb:2]: the group has_many :students_groups (with plural form in the end, right?)

i've changed that

File: app/models/students_group.rb (hunk 2 above)
Comment (mikeinflunity) [reply app/models/students_group.rb:3]: ok corrected that

File: db/migrate/20200630120116_create_groups.rb (hunk 3 above)
Comment (mikeinflunity) [reply db/migrate/20200630120116_create_groups.rb:5]: we can add the reference when both models are merged to the master

File: db/schema.rb (hunk 4 above)
Comment (mikeinflunity) [reply db/schema.rb:22]: yes that's probably what happen, should we delete this line or make a new migration for the  groups

File: app/models/group.rb (hunk 1 above)
Comment (arieivs) [reply app/models/group.rb]: right! plural indeed

**Pull request merged**